boot/freeldr/freeldr/lib/fs/fs.c

```diff
@@ -43,162 +43,221 @@ typedef struct tagDEVICE
 {
     LIST_ENTRY ListEntry;
     const DEVVTBL* FuncTable;
-    CHAR* Prefix;
+    PSTR DeviceName;
     ULONG DeviceId;
     ULONG ReferenceCount;
 } DEVICE;
 
 static FILEDATA FileData[MAX_FDS];
 static LIST_ENTRY DeviceListHead;
 
+#define IS_VALID_FILEID(FileId) \
+    ((FileId) < _countof(FileData) && FileData[(FileId)].FuncTable)
+
+typedef const DEVVTBL* (*PFS_MOUNT)(ULONG DeviceId);
+
+PFS_MOUNT FileSystems[] =
+{
+#ifndef _M_ARM
+    IsoMount,
```

Comment:
Unblocking ISOMOUNT is one of the things i do in my ARM32 PR, it doesnt even need changes to build on its own last i checked.

Reply:
What do you mean by "unblocking"? You mean, removing this `#ifndef _M_ARM` here (and in the `iso.c` file) ?
Iirc. it could have been removed from ARM32 builds because that platform didn't use CD drives, where ISO9660 is the main thing.

Reply:
this is just a note for me to remove the arm check later.